src/utils/github.js

```diff
@@ -54,11 +45,7 @@ export async function createRepo (githubOrg, githubRepo, templateOrg, templateRe
     console.log(`✅ Created code repository at https://github.com/${githubOrg}/${githubRepo} from template ${templateOrg}/${templateRepo}`)
     await modifyFstab(githubOrg, githubRepo, templateRepo)
     await modifySidekickConfig(githubOrg, githubRepo)
-
-    // aem-boilerplate-commerce is on config service, so for scaffolded repos we need to create a local config.json
-    if (templateRepo === 'aem-boilerplate-commerce') {
-      await createLocalCommerceConfig(githubOrg, githubRepo, templateOrg, templateRepo)
-    }
+    await createLocalCommerceConfig(githubOrg, githubRepo, templateOrg, templateRepo)
```

Comment:
Now we always create this local commerce config - it is how boilerplate+ should work.